matiasbenedetto/google-fonts-to-wordpress-collection
# Run format:json script after running files and previews scripts

## What?
- Adds `format:json` script to format all the JSON files from releases.  
- Runs `format:json` script after running files and previews scripts.
- Format the only JSON file missing the `wp-scripts` formatting style.


## Why?
To format the JSON files when their data is updated.

## Testing instructions

Run `npm run previews` to generate the font previews.
As a result, you should not have any diff because the generated data (preview SVG files and data JSON files) should be the same as the data existing in this branch.
